htdocs/class/libraries/vendor/xoops/xmf/src/Database/TableLoad.php

```diff
@@ -130,7 +130,7 @@ public static function countRows($table, $criteria = null)
         $prefixedTable = $db->prefix($table);
         $sql = 'SELECT COUNT(*) as `count` FROM ' . $prefixedTable . ' ';
         if (isset($criteria) && is_subclass_of($criteria, '\CriteriaElement')) {
-            /* @var  $criteria \CriteriaCompo */
+            /* @var  \CriteriaCompo $criteria */
```

Comment:
No direct changes are to be made in the libraries directory. The build will overwrite these changes, making this pointless.